zap/src/output/luau/client.rs

```diff
@@ -70,6 +70,52 @@ impl<'src> ClientOutput<'src> {
 		}
 	}
 
+	fn push_event_loop(&mut self) {
+		self.push_line("");
+
+		let delta_time = if self.config.manual_event_loop {
+			self.config.casing.with("DeltaTime", "deltaTime", "delta_time")
+		} else {
+			"dt"
+		};
+
+		if self.config.manual_event_loop {
+			let send_events = self.config.casing.with("SendEvents", "sendEvents", "send_events");
+
+			self.push_line(&format!("local function {send_events}({delta_time}: number)"));
+		} else {
+			self.push_line(&format!("RunService.Heartbeat:Connect(function({delta_time})"));
```

Comment:
And hardcode in deltatime here
```suggestion
			self.push_line(&format!("RunService.Heartbeat:Connect(function(dt)"));
```